lib/matplotlib/patches.py

```diff
@@ -797,15 +814,17 @@ def set_bounds(self, *args):
             l, b, w, h = args[0]
         else:
             l, b, w, h = args
-        self._x = l
-        self._y = b
+        self._x0 = l
+        self._y0 = b
         self._width = w
         self._height = h
+        self._update_x1()
+        self._update_y1()
         self.stale = True
 
     def get_bbox(self):
-        return transforms.Bbox.from_bounds(self._x, self._y,
-                                           self._width, self._height)
+        return transforms.Bbox.from_extents(self._x0, self._y0,
```

Comment:
Doesn't this need to use the unit conversions, as in _update_patch_transform()?

Reply:
I guess it probably does, but no-one every added it originally. I'll add it in another commit.